### ISSUE TITLE
EvolGAN: Evolutionary Generative Adversarial Networks

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#126]
# TL;DR
Research to improve the quality of generated images by optimizing latent variables using evolutionary strategies. Using an evaluator that predicts human ratings (Koncept512) to measure the quality of the image while guiding the latent variable to a better place. Resulted in better images being generated than before optimization.

![Fig  1  For illustration , random images generated using StyleGAN2 (left) and Evol-](https://user-images.githubusercontent.com/24524018/95158564-def62080-07d6-11eb-8e6d-ad2ea961ad50.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.13311

# Submission Dates(yyyy/mm/dd)
2020/09/28

# Authors and institutions

# Methods

# Results

# Comments
